Remotion/Web/Development.WebTesting/WebDriver/Factories/Edge/EdgeBrowserFactory.cs

```diff
@@ -55,7 +55,10 @@ public IBrowserSession CreateBrowser (DriverConfiguration configuration)
 
       var extendedEdgeOptions = _edgeConfiguration.CreateEdgeOptions();
       if (configuration.Headless)
-        extendedEdgeOptions.AddArgument("headless");
+        extendedEdgeOptions.AddArgument("headless=new");
+
+      // turn off sidebar which kills some tests.
+      extendedEdgeOptions.AddArgument("disable-features=msHubApps");
```

Comment:
Do you want to keep the distinct commits for the individual feature toggles? if so, please update the commit messages.